tests/configs/tls13-only.h

```diff
@@ -25,10 +25,13 @@
 #define MBEDTLS_SSL_PROTO_TLS1_3
 #define MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
 
+#undef MBEDTLS_SSL_DTLS_CONNECTION_ID
```

Comment:
Are these configs ever built, e.g. by copying them to `include/mbedtls/mbedtls_config.h` and building, or are they just untested examples?